test/gst-vaapi/util.py

```diff
@@ -20,8 +20,7 @@ def get_supported_format_map():
     "I420"  : ("i420", "I420"),
     "NV12"  : ("nv12", "NV12"),
     "YV12"  : ("yv12", "YV12"),
-    "AYUV"  : ("ayuv", "AYUV"),
-    "VUYA"  : ("vuya", "VUYA"),
+    "AYUV"  : ("vuya", "VUYA"),
```

Comment:
Please put a comment here that states we use microsoft's definition of AYUV (and a link to that definition), please.

Reply:
updated